internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -39,34 +40,36 @@ func New(sess *session.Session) *ELBV2 {
 	}
 }
 
-// ListenerRuleHostHeaders returns all the host headers for a listener rule.
-func (e *ELBV2) ListenerRuleHostHeaders(ruleARN string) ([]string, error) {
+// ListenerRulesHostHeaders returns all the host headers for all listener rules.
+func (e *ELBV2) ListenerRulesHostHeaders(ruleARNs []string) ([]string, error) {
 	resp, err := e.client.DescribeRules(&elbv2.DescribeRulesInput{
-		RuleArns: aws.StringSlice([]string{ruleARN}),
+		RuleArns: aws.StringSlice(ruleARNs),
 	})
 	if err != nil {
-		return nil, fmt.Errorf("get listener rule for %s: %w", ruleARN, err)
+		return nil, fmt.Errorf("get listener rule for %s: %w", english.WordSeries(ruleARNs, "and"), err)
 	}
 	if len(resp.Rules) == 0 {
-		return nil, fmt.Errorf("cannot find listener rule %s", ruleARN)
+		return nil, fmt.Errorf("cannot find listener rule %s", english.WordSeries(ruleARNs, "and"))
 	}
-	rule := resp.Rules[0]
 	hostHeaderSet := make(map[string]bool)
```

Comment:
nit: what do you think about
```suggestion
       exists := struct{}{}
	hostHeaderSet := make(map[string]struct{})
	...
	for _, value := range condition.Values {
					hostHeaderSet[aws.StringValue(value)] = exists
				}
```
I can think of two benefit of this - 
1. If we use `bool`, then it's ambiguous what does `hostHeaderSet["this"] = false` means.
2. `struct{}` takes 0 byte